compiler/Makefile

```diff
@@ -133,10 +134,19 @@ generate_example_in_container:
 test_in_container: ${BUILDER_NAME}\:${BUILDER_TAG}.image.exists
 	$(call run_in_container, make test)
 
-.PHONY: generate_code
+
 generate_code:
+	$(MAKE) run_generate_code || { $(MAKE) run_backup ; exit 1; }
```

Comment:
Looks like run_backup is actually restore from backup ?

The source of truth here is the filesystem. Which seems a little flaky. Lets discuss this on a call.

Reply:
@PavitraS Can you add comment explaining how this line works? It's a little bit tricky to understand what is going on.

Reply:
don't need this code anymore. removed.